examples/campaign_management/create_experiment.rb

```diff
@@ -86,47 +85,24 @@ def create_experiment_arms(client, customer_id, base_campaign_id, experiment)
   response = client.service.experiment_arm.mutate_experiment_arms(
     customer_id: customer_id,
     operations: operations,
+    # We want to fetch the draft campaign IDs from the treatment arm, so the
+    # easiest way to do that is to have the response return the newly created
+    # entities.
+    response_content_type: :MUTABLE_RESOURCE,
   )
 
   # Results always return in the order that you specify them in the request.
-  # Since we created the treatment arm last, it will be the last result.  If
-  # you don't remember which arm is the treatment arm, you can always filter
-  # the query in the next section with `experiment_arm.control = false`.
-  control_arm = response.results.first.resource_name
-  treatment_arm = response.results.last.resource_name
+  # Since we created the treatment arm last, it will be the last result.
+  control_arm_result = response.results.first
+  treatment_arm_result = response.results.last
 
-  puts "Created control arm with resource name #{control_arm}."
-  puts "Created treatment arm with resource name #{treatment_arm}."
+  puts "Created control arm with resource name #{control_arm_result.resource_name}."
+  puts "Created treatment arm with resource name #{treatment_arm_result.resource_name}."
 
-  treatment_arm
+  treatment_arm_result.experiment_arm.in_design_campaigns.first
```

Comment:
Is the `in_design_campaigns` attribute mutable? If not, it won't be returned even with `response_content_type = MUTABLE_RESOURCE`.

Reply:
It is not mutable, and it is returned anyway for this feature specifically for this specific use case (to avoid a followup request which would need to be done pretty much every time you ever make an experiment arm).